lms/routes.py

```diff
@@ -105,3 +105,5 @@ def includeme(config):
     config.add_route("admin.index", "/admin/")
     config.add_route("admin.instances", "/admin/instances/")
     config.add_route("admin.instance", "/admin/instance/{consumer_key}/")
+
+    config.add_route("lti.oidc", "/lti/1.3/oidc")
```

Comment:
This doesn't belong to /api/lti, that seems to be reserved now for LTI APIS (eg grading APIs).

I think starting a new "namespace" here makes sense. Arguably `content_item_selection` and `lti_launches` should also be in /lti but I'll leave them as they are for now.